grapher/text/parser.ts

```diff
@@ -305,7 +305,7 @@ const markdownLinkParser: (r: MdParser) => P.Parser<MarkdownLink> = (
                 .atLeast(1),
         ],
         P.string("]("),
-        ["href", P.regex(urlRegex)],
+        ["href", P.alt(P.regex(/\/[\w\-]+/), P.regex(urlRegex))],
```

Comment:
Relative URLs aren't quite reliable since we can have iframeless embeds on [sdg-tracker.org](https://sdg-tracker.org).

Not sure how to handle that – what do you think?

- We could discourage relative links by not parsing them
- We could prepend `https://ourworldindata.org` to relative URLs
- Do nothing, leave it as it is now. This is maybe also fine, and in terms of reusability of the Grapher it's a better option.